### PR TITLE
Added langs : es, fr, nl, jp, ru, it and updated already existing ones

### PR DESCRIPTION
**Pull request by maximeurban**
### **Added new lang files :** 
Spanish
French
Dutch
Japanese
Russian

### **Updated lang files :**
German
Polish
Deutch
Simplified Chinese
Brazilian Portuguese / Portugese

### CODE REVIEW

File: src/main/resources/assets/viafabric/lang/pt_br.json [hunk 1]
```diff
@@ -3,5 +3,12 @@
   "gui.enable_client_side_button": "Habilitar ViaVersion",
   "gui.enable_client_side.question": "Você tem certeza de que deseja habilitar o modo client-side?",
   "gui.enable_client_side.warning": "Não posso garantir que este mod seja permitido em todos os (ou mesmo em quaisquer) servidores. Esse poderá causar problemas com plugins anti-trapaça. USE POR SUA PRÓPRIA CONTA E RISCO.",
-  "gui.enable_client_side.enable": "Habilitar"
+  "gui.enable_client_side.enable": "Habilitar",
+  "gui.viafabric_config.title": "Configurações de ViaFabric",
+  "gui.client_side.enable": "Habilitar o lado do cliente",
+  "gui.client_side.disable": "Desactivar o lado do cliente",
```
Comment (creeper123123321) [inline src/main/resources/assets/viafabric/lang/pt_br.json:9]: "Desactivar" is pt-pt

**Review by creeper123123321 — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/viafabric/lang/pt_br.json (hunk 1 above)
Comment (maximeurban) [reply src/main/resources/assets/viafabric/lang/pt_br.json:9]: It's actually both Spanish and Portuguese

File: src/main/resources/assets/viafabric/lang/de_de.json [hunk 2]
```diff
@@ -3,5 +3,12 @@
   "gui.enable_client_side_button": "ViaVersion aktivieren",
   "gui.enable_client_side.question": "Bist du dir sicher, dass du den clientseitigen Modus aktivieren möchtest?",
   "gui.enable_client_side.warning": "Ich kann nicht garantieren, dass diese Mod auf jedem (oder überhaupt einem) Server erlaubt ist. Diese Mod könnte eventuell Probleme mit Anti-Cheat Plugins hervorrufen. VERWENDUNG AUF EIGENE GEFAHR.",
-  "gui.enable_client_side.enable": "Aktivieren"
+  "gui.enable_client_side.enable": "Aktivieren",
+  "gui.viafabric_config.title": "ViaFabric-Konfigurationen",
+  "gui.client_side.enable": "Client-seitig aktivieren",
+  "gui.client_side.disable": "Client-Seite deaktivieren",
+  "gui.ping_version.translated": "VIA: %s",
+  "gui.hide_via_button.enable": "Schaltfläche VIA ausblenden",
```
Comment (Jo0001) [inline src/main/resources/assets/viafabric/lang/de_de.json:11]: This should be _VIA Button ausblenden_

File: src/main/resources/assets/viafabric/lang/de_de.json [hunk 3]
```diff
@@ -3,5 +3,12 @@
...
+  "gui.hide_via_button.disable": "Schaltfläche VIA anzeigen",
```
Comment (Jo0001) [inline src/main/resources/assets/viafabric/lang/de_de.json:12]: And this  _VIA Button anzeigen_

File: src/main/resources/assets/viafabric/lang/de_de.json [hunk 4]
```diff
@@ -3,5 +3,12 @@
   "gui.enable_client_side_button": "ViaVersion aktivieren",
   "gui.enable_client_side.question": "Bist du dir sicher, dass du den clientseitigen Modus aktivieren möchtest?",
   "gui.enable_client_side.warning": "Ich kann nicht garantieren, dass diese Mod auf jedem (oder überhaupt einem) Server erlaubt ist. Diese Mod könnte eventuell Probleme mit Anti-Cheat Plugins hervorrufen. VERWENDUNG AUF EIGENE GEFAHR.",
-  "gui.enable_client_side.enable": "Aktivieren"
+  "gui.enable_client_side.enable": "Aktivieren",
+  "gui.viafabric_config.title": "ViaFabric-Konfigurationen",
+  "gui.client_side.enable": "Client-seitig aktivieren",
```
Comment (Jo0001) [inline src/main/resources/assets/viafabric/lang/de_de.json:8]: This should be _Clientseitigen Modus aktivieren_

File: src/main/resources/assets/viafabric/lang/de_de.json [hunk 5]
```diff
@@ -3,5 +3,12 @@
...
+  "gui.client_side.disable": "Client-Seite deaktivieren",
```
Comment (Jo0001) [inline src/main/resources/assets/viafabric/lang/de_de.json:9]: And _Clientseitigen Modus deaktivieren_

**Review by Jo0001 — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/viafabric/lang/ru_ru.json [hunk 6]
```diff
@@ -0,0 +1,14 @@
+{
+  "gui.protocol_version_field.name": "Версия протокола",
+  "gui.enable_client_side_button": "Включить ViaVersion",
+  "gui.enable_client_side.question": "Вы уверены, что хотите включить режим на стороне клиента?",
+  "gui.enable_client_side.warning": "Я не могу гарантировать, что этот мод разрешен на каждом (или даже любом) сервере. Этот мод может вызвать проблемы с анти-разогревающими плагинами. ИСПОЛЬЗОВАНИЕ НА СВОЙ СТРАХ И РИСК.",
+  "gui.enable_client_side.enable": "Включить",
+  "gui.viafabric_config.title": "ViaFabric Конфигурации",
```
Comment (mihapain1) [inline src/main/resources/assets/viafabric/lang/ru_ru.json:7]: `"gui.viafabric_config.title": "Настройка ViaFabric ",`

File: src/main/resources/assets/viafabric/lang/ru_ru.json [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+  "gui.client_side.enable": "Разрешите клиентскому отделу",
```
Comment (mihapain1) [inline src/main/resources/assets/viafabric/lang/ru_ru.json:8]: `"gui.client_side.enable": "Включить на стороне клиента",`

File: src/main/resources/assets/viafabric/lang/ru_ru.json [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+  "gui.client_side.disable": "Отключить сторону клиента",
```
Comment (mihapain1) [inline src/main/resources/assets/viafabric/lang/ru_ru.json:9]: `"gui.client_side.disable": "Выключить на стороне клиента",`

File: src/main/resources/assets/viafabric/lang/de_de.json (hunk 4 above)
Comment (maximeurban) [reply src/main/resources/assets/viafabric/lang/de_de.json:8]: yes you added mod to the traduction but in the original text there wasn't the word mod

File: src/main/resources/assets/viafabric/lang/de_de.json (hunk 5 above)
Comment (maximeurban) [reply src/main/resources/assets/viafabric/lang/de_de.json:9]: yes you added mod to the traduction but in the original text there wasn't the word mod

File: src/main/resources/assets/viafabric/lang/de_de.json (hunk 3 above)
Comment (maximeurban) [reply src/main/resources/assets/viafabric/lang/de_de.json:12]: means the same to me

File: src/main/resources/assets/viafabric/lang/de_de.json (hunk 2 above)
Comment (maximeurban) [reply src/main/resources/assets/viafabric/lang/de_de.json:11]: means the same to me

File: src/main/resources/assets/viafabric/lang/ru_ru.json [hunk 9]
```diff
@@ -0,0 +1,14 @@
+{
+  "gui.protocol_version_field.name": "Версия протокола",
+  "gui.enable_client_side_button": "Включить ViaVersion",
+  "gui.enable_client_side.question": "Вы уверены, что хотите включить режим на стороне клиента?",
+  "gui.enable_client_side.warning": "Я не могу гарантировать, что этот мод разрешен на каждом (или даже любом) сервере. Этот мод может вызвать проблемы с анти-разогревающими плагинами. ИСПОЛЬЗОВАНИЕ НА СВОЙ СТРАХ И РИСК.",
```
Comment (mihapain1) [inline src/main/resources/assets/viafabric/lang/ru_ru.json:5]: `"gui.enable_client_side.warning": "Я не могу гарантировать, что этот мод разрешен на любом сервере. Он может вызвать проблемы с анти-чит плагинами. ИСПОЛЬЗОВАТЬ НА СВОЙ СТРАХ И РИСК.",`

File: src/main/resources/assets/viafabric/lang/pt_br.json (hunk 1 above)
Comment (creeper123123321) [reply src/main/resources/assets/viafabric/lang/pt_br.json:9]: I meant that "desactivar" isn't used in brazilian portuguese, it is usually used in european portuguese

File: src/main/resources/assets/viafabric/lang/ru_ru.json [hunk 10]
```diff
@@ -0,0 +1,14 @@
+{
+  "gui.protocol_version_field.name": "Версия протокола",
+  "gui.enable_client_side_button": "Включить ViaVersion",
+  "gui.enable_client_side.question": "Вы уверены, что хотите включить режим на стороне клиента?",
```
Comment (mihapain1) [inline src/main/resources/assets/viafabric/lang/ru_ru.json:4]: `"gui.enable_client_side.question": "Вы уверены, что хотите включить мод на стороне клиента?",`

**Review by mihapain1 — CHANGES_REQUESTED** [review]

**Review by mihapain1 — APPROVED** [review]

File: src/main/resources/assets/viafabric/lang/pt_br.json (hunk 1 above)
Comment (maximeurban) [reply src/main/resources/assets/viafabric/lang/pt_br.json:9]: okay i changed it anyways

**Review by Jo0001 — APPROVED** [review]

File: src/main/resources/assets/viafabric/lang/ru_ru.json [hunk 11]
```diff
@@ -0,0 +1,14 @@
+{
+  "gui.protocol_version_field.name": "Настройка ViaFabric",
```
Comment (mihapain1) [inline src/main/resources/assets/viafabric/lang/ru_ru.json:2]: "gui.protocol_version_field.name": "Версия протокола",

File: src/main/resources/assets/viafabric/lang/ru_ru.json [hunk 12]
```diff
@@ -0,0 +1,14 @@
...
+  "gui.enable_client_side_button": "Включить ViaFabric",
+  "gui.enable_client_side.question": "Вы уверены, что хотите включить мод на стороне клиента?",
+  "gui.enable_client_side.warning": "Я не могу гарантировать, что этот мод разрешен на любом сервере. Он может вызвать проблемы с анти-чит плагинами. ИСПОЛЬЗОВАТЬ НА СВОЙ СТРАХ И РИСК.",
+  "gui.enable_client_side.enable": "Включить",
+  "gui.viafabric_config.title": "ViaFabric Конфигурации",
```
Comment (mihapain1) [inline src/main/resources/assets/viafabric/lang/ru_ru.json:7]: `"gui.protocol_version_field.name": "Настройка ViaFabric",`

**Review by mihapain1 — CHANGES_REQUESTED** [review]
typo ctrl+c ctrl+v

File: src/main/resources/assets/viafabric/lang/ru_ru.json (hunk 12 above)
Comment (maximeurban) [reply src/main/resources/assets/viafabric/lang/ru_ru.json:7]: I think there's an error there

Comment (maximeurban) [reply src/main/resources/assets/viafabric/lang/ru_ru.json:7]: oh wait it's just the wrong field you meant config title

**Review by mihapain1 — APPROVED** [review]

File: src/main/resources/assets/viafabric/lang/de_de.json [hunk 13]
```diff
@@ -3,5 +3,12 @@
   "gui.enable_client_side_button": "ViaVersion aktivieren",
   "gui.enable_client_side.question": "Bist du dir sicher, dass du den clientseitigen Modus aktivieren möchtest?",
   "gui.enable_client_side.warning": "Ich kann nicht garantieren, dass diese Mod auf jedem (oder überhaupt einem) Server erlaubt ist. Diese Mod könnte eventuell Probleme mit Anti-Cheat Plugins hervorrufen. VERWENDUNG AUF EIGENE GEFAHR.",
-  "gui.enable_client_side.enable": "Aktivieren"
+  "gui.enable_client_side.enable": "Aktivieren",
+  "gui.viafabric_config.title": "ViaFabric-Konfigurationen",
+  "gui.client_side.enable": "Clientseitigen Modus aktivieren",
```
Comment (creeper123123321) [inline src/main/resources/assets/viafabric/lang/de_de.json:8]: text overflows

File: src/main/resources/assets/viafabric/lang/ru_ru.json [hunk 14]
```diff
@@ -0,0 +1,14 @@
+{
+  "gui.protocol_version_field.name": "Версия протокола",
+  "gui.enable_client_side_button": "Включить ViaFabric",
+  "gui.enable_client_side.question": "Вы уверены, что хотите включить мод на стороне клиента?",
+  "gui.enable_client_side.warning": "Я не могу гарантировать, что этот мод разрешен на любом сервере. Он может вызвать проблемы с анти-чит плагинами. ИСПОЛЬЗОВАТЬ НА СВОЙ СТРАХ И РИСК.",
+  "gui.enable_client_side.enable": "Включить",
+  "gui.viafabric_config.title": "Настройка ViaFabric",
+  "gui.client_side.enable": "Включить на стороне клиент",
+  "gui.client_side.disable": "Выключить на стороне клиента",
```
Comment (creeper123123321) [inline src/main/resources/assets/viafabric/lang/ru_ru.json:9]: this seems to overflow too

File: src/main/resources/assets/viafabric/lang/de_de.json (hunk 13 above)
Comment (Gerrygames) [reply src/main/resources/assets/viafabric/lang/de_de.json:8]: "Clientseitig aktivieren" might work

**Pull request merged**